roles/zuul/files/jobs/hoist.yaml

```diff
@@ -22,3 +23,4 @@
       - shell: |
           ./tests/signed-off-by-test.sh
           python tests/layout-checks.py job-list.txt
+          ./tests/shellcheck-test.sh
```

Comment:
We certainly can. I added this test to the end because I'm not entirely sure how this mechanism works. In Travis CI, we are running the tests in parallel, but I think they happen sequentially here. If they are sequential, I think it makes sense to add this check to the end?